### ISSUE TITLE
rule_sshd_do_not_permit_user_env: Check and remidiations set "PermitUserEnvironment yes"

### ISSUE THREAD

**Issue by jaedolph** [ComplianceAsCode/content#5902]
#### Description of problem:
Description for xccdf_org.ssgproject.content_rule_sshd_do_not_permit_user_env check tells the user to set `PermitUserEnvironment no`:
> To ensure users are not able to override environment variables of the SSH daemon, add or correct the following line in `/etc/ssh/sshd_config`: 
> ```
> PermitUserEnvironment no
 > ```

However, when this is set in `/etc/ssh/sshd_config` the check still fails.

When `PermitUserEnvironment` is set to `yes` the check passes, which contradicts the description of the remediation.

Also, the bash and ansible remediations both set `PermitUserEnvironment yes`.

#### SCAP Security Guide Version:
0.1.46-11.el7

#### Operating System Version:
Red Hat Enterprise Linux Server release 7.7 (Maipo)

#### Steps to Reproduce:

1. Set `PermitUserEnvironment no` in `/etc/ssh/sshd_config`
2. Run `oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_e8 --report report.html /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml`
3. Set  `PermitUserEnvironment yes` in `/etc/ssh/sshd_config`
4. Run `oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_e8 --report report.html /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml`

#### Actual Results:
with `PermitUserEnvironment no`:
```
Title   Do Not Allow SSH Environment Options
Rule    xccdf_org.ssgproject.content_rule_sshd_do_not_permit_user_env
Ident   CCE-27363-1
Result  fail
```
with `PermitUserEnvironment yes`
```
Title   Do Not Allow SSH Environment Options
Rule    xccdf_org.ssgproject.content_rule_sshd_do_not_permit_user_env
Ident   CCE-27363-1
Result  fail
```

#### Expected Results:
with `PermitUserEnvironment no`:
```
Title   Do Not Allow SSH Environment Options
Rule    xccdf_org.ssgproject.content_rule_sshd_do_not_permit_user_env
Ident   CCE-27363-1
Result  pass
```
with `PermitUserEnvironment yes`:
```
Title   Do Not Allow SSH Environment Options
Rule    xccdf_org.ssgproject.content_rule_sshd_do_not_permit_user_env
Ident   CCE-27363-1
Result  fail
```

**Comment by ggbecker** [comment]
Thank you for the report. This issue has been already fixed in: https://github.com/ComplianceAsCode/content/pull/5087

And it's part of releases equal or greater than 0.1.48.

**Comment by jaedolph** [comment]
Thanks @ggbecker. I guess this patch hasn't made its way in to the RHEL7 release yet.

**Comment by ggbecker** [comment]
> Thanks @ggbecker. I guess this patch hasn't made its way in to the RHEL7 release yet.

No, it hasn't. It will be fixed in the next minor release (7.9).
